data/nation/congolesebrazzaville.ruleset

```diff
@@ -37,7 +37,7 @@ init_buildings=""
 init_units=""
 
 conflicts_with = "kongo"
-civilwar_nations = "congolese", "central african"
+civilwar_nations = "congolese", "central_african",
```

Comment:
Almost there, but still one comma too much.

Reply:
Man I thought I got that.